scripts/check_docs_formatted.py

```diff
@@ -238,6 +248,28 @@ def format_file(file: Path, error_known: bool, args: argparse.Namespace) -> int:
     return 0
 
 
+def find_backticked_shortcut_links(
+    path: Path, all_config_names: dict[str, object]
+) -> set[str]:
+    """Check for links of the form ```[`foobar`]```.
```

Comment:
I find the triple ` somewhat confusing because it wasn't clear to me if it actually matches three backticks or not. I'd probably use regualr quotes here or a colon
```suggestion
    """Check for links of the form: [`foobar`].
```

Reply:
Can you say more about what docstring conventions you mean? It might be that this is a convention that I'm not familiar with